.github/workflows/preview.yml

```diff
@@ -7,8 +7,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
See https://github.com/actions/checkout#checkout-pull-request-head-commit-instead-of-merge-commit